src/test/java/com/st/utopia/cancellationservice/dao/TicketDaoTest.java

```diff
@@ -0,0 +1,53 @@
+package com.st.utopia.cancellationservice.dao;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.st.utopia.cancellationservice.model.Ticket;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TicketDaoTest {
+	
+	@Autowired
+    private TicketDao ticketdao;
+
+    @Test
+    public void createTicket() {
+    	LocalDateTime dout = LocalDateTime.parse("28-01-2019 12:00",DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm"));
+    	Ticket t = new Ticket(1 , 11, 'B',11, 1, 1500, dout,"jhytrfgr");
+    	ticketdao.save(t);
+    	assertTrue(t.getBookingid().equals("jhytrfgr"));

```

Comment:
When you create an object yourself and hand it to the database via `save`, the original object still remains intact, so here all you're `assert`ing is that the object you created has the field value you gave it, not that it actually made it to the database.

Reply:
Yes, you will need to make another call to double check